packages/admin/src/FormSaveSplitButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { FORM_ERROR } from "final-form";
+import * as React from "react";
+import { PropsWithChildren } from "react";
+import { useForm, useFormState } from "react-final-form";
+import { FormattedMessage } from "react-intl";
+
+import { SaveButton } from "./common/buttons/save/SaveButton";
+import { SplitButton } from "./common/buttons/split/SplitButton";
+import { useStackApi } from "./stack/Api";
+
+export interface FormSaveButtonProps {
+    localStorageKey?: string;
+    saving: boolean;
+    hasErrors: boolean;
+}
+
+export const FormSaveSplitButton = ({ saving, hasErrors, localStorageKey }: PropsWithChildren<FormSaveButtonProps>) => {
```

Comment:
Should we rename this component to `FinalFormSaveSplitButton`? Doing so would be more in line with our other `final-form` related components.